src/pages/ProfilePage.js

```diff
@@ -200,15 +201,16 @@ function ProfilePage(props) {
                             ) : null}
                             {shouldShowLocalTime && <AutoUpdateTime timezone={timezone} />}
                         </View>
-                        {!isCurrentUser && Boolean(login) && (
-                            <MenuItem
-                                title={`${props.translate('common.message')}${displayName}`}
-                                icon={Expensicons.ChatBubble}
-                                onPress={() => Report.navigateToAndOpenReport([login])}
-                                wrapperStyle={styles.breakAll}
-                                shouldShowRightIcon
-                            />
-                        )}
+                        {!isCurrentUser &&
+                            !Session.isAnonymousUser(
+                                <MenuItem
+                                    title={`${props.translate('common.message')}${displayName}`}
+                                    icon={Expensicons.ChatBubble}
+                                    onPress={() => Report.navigateToAndOpenReportWithAccountIDs([accountID])}
+                                    wrapperStyle={styles.breakAll}
+                                    shouldShowRightIcon
+                                />,
```

Comment:
```suggestion
                                />
```

This needs to go to as well due to the change above.